assets/test/Line.test.ts

```diff
@@ -115,46 +115,81 @@ test('Shows batch mode buttons when not read-only', () => {
   expect(screen.getByText('All to off')).toBeInTheDocument();
 });
 
-test.each([['Silver'], ['Busway']])(
-  'Does not show headway batch mode button',
-  (line) => {
-    const now = Date.now();
-    const signs = {};
-
-    const currentTime = now + 2000;
-    const signConfigs = {};
-    const setConfigs = () => true;
-    const readOnly = false;
-    const configuredHeadways = {};
-    const setConfiguredHeadways = () => true;
-
-    render(
-      React.createElement(
-        Line,
-        {
-          alerts: {},
-          signs,
-          currentTime,
-          line,
-          signConfigs,
-          setConfigs,
-          readOnly,
-          configuredHeadways,
-          setConfiguredHeadways,
-          chelseaBridgeAnnouncements: 'off',
-          setChelseaBridgeAnnouncements: () => {},
-          signGroups: {},
-          setSignGroups: () => {},
-        },
-        null,
-      ),
-    );
-
-    expect(screen.getByText('All to auto')).toBeInTheDocument();
-    expect(screen.queryByText('All to headway')).toBeNull();
-    expect(screen.getByText('All to off')).toBeInTheDocument();
-  },
-);
+test('Does not show headway batch mode button', () => {
```

Comment:
Since we removed the `test.each` that made it clearer up-front what the relevant part of the test is, maybe we could make this like "Does not show headway batch mode for busways" (and the below "Shows headway batch mode for Silver Line").